src/bundle/Resources/views/themes/admin/content/tab/locations/panel_swap.html.twig

```diff
@@ -10,7 +10,7 @@
     {{ 'tab.locations.swap_with_another'|trans()|desc('Swap the Content item at this Location with another') }}
     {{ form_widget(form.swap, {'attr': {
         'disabled': not can_swap,
-        'class': 'btn btn-secondary btn--udw-swap ml-5',
+        'class': 'btn ibexa-btn ibexa-btn--secondary btn--udw-swap ml-5',
```

Comment:
The class `btn--udw-swap` is only used in
`/src/bundle/Resources/public/js/scripts/udw/swap.js` maybe we could replace `btn--udw-swap` with `ibexa--udw-swap`?